catalog/ui/catalog-ui-search/src/main/webapp/js/controllers/ol.layerCollection.controller.js

```diff
@@ -117,6 +119,78 @@ define(['underscore',
                 if (initObj.url && initObj.url.indexOf('/tile/{z}/{y}/{x}') === -1) {
                     initObj.url = initObj.url + '/tile/{z}/{y}/{x}';
                 }
+            } else if (typeStr === 'WMT') {
+                /* If tileMatrixSetID is present (Cesium WMTS keyword) set matrixSet (OpenLayers WMTS keyword) */
+                if (initObj.tileMatrixSetID) {
+                    initObj.matrixSet = initObj.tileMatrixSetID;
+                }
+
+                /* We must mock the tileGrid in order to prevent errors in OpenLayers before the GetCapabilities request returns */
+                initObj.tileGrid = {
```

Comment:
Let's store the mock in a variable `mockTileGrid` at the top, next to `imageryProviderTypes`.